packages/dapp/public/index.html

```diff
@@ -12,6 +12,32 @@
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
     <title>ETHRio Stays Dapp</title>
+    <style>
```

Comment:
custom styles should not be added directly into the index.html.
If you want to add a custom style please use this file ./src/reset.css
this file is loading at the very first application loading stage